rosbag2_storage_mcap/src/mcap_storage.cpp

```diff
@@ -47,7 +47,7 @@
 #include <utility>
 #include <vector>
 #ifdef ROSBAG2_STORAGE_MCAP_HAS_STORAGE_FILTER_TOPIC_REGEX
-#include <regex>
+  #include <regex>
```

Comment:
does it work to include spaces before the #?

Reply:
See `.clang-format` https://github.com/ros-tooling/rosbag2_storage_mcap/pull/73/files#diff-f064a7203feb3ead7ec4759c19472ef142aaaf2c2fcd526ee719575d7b1e95a0R20 - this package uses `BeforeHash` setting to better match the existing rosbag2 source style